edge-modules/MetricsCollector/Program.cs

```diff
@@ -29,6 +32,8 @@ private static async Task<int> MainAsync()
             try
             {
                 moduleClient = await ModuleClient.CreateFromEnvironmentAsync(transportSettings);
+                Dictionary<string, string> testSpecificTags = await GetTestSpecificTagsFromTwin(moduleClient);

```

Comment:
why do you want to get additional metric tags from Twin? requirements from customer?